src/libraries/Microsoft.Extensions.DependencyModel/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+[assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
```

Comment:
This issue talks about crypto. Is it correct issue to reference here?

I would expect Microsoft.Extensions.DependencyModel tests to have no or few dependencies on crypto.

Reply:
For numerous amounts of tests, we are throwing PNSE b/c of crypto.  

This method is the culprit:

https://github.com/dotnet/cli-migrate/blob/5395650fe2a118fb8d5e7d1475a0b684d969e908/src/Microsoft.DotNet.ProjectJsonMigration/Microsoft.DotNet.Internal.ProjectModel/DependencyContextBuilder.cs#L65-L87

Reply:
Thanks for the link